PinpointKit/PinpointKit/Sources/Core/Editing/EditImageViewController.swift

```diff
@@ -564,6 +571,8 @@ public final class EditImageViewController: UIViewController, UIGestureRecognize
         let removeAnnotationView = {
             self.endEditingTextView()
             annotationView.removeFromSuperview()
+            
+            self.informDelegateOfChange()
```

Comment:
We say `change` here and `changes` in the delegate method. I think we should standardize on one of those.